### PR TITLE
test(x/swaprouter): TestCreatePool (merge)

### PR DESCRIPTION
**Pull request by p0mvn**
Closes: #XXX

## What is the purpose of the change

Merging test for swaprouter's `CreatePool`

### CODE REVIEW

File: x/swaprouter/create_pool_test.go [hunk 1]
```diff
@@ -62,7 +63,7 @@ func (suite *KeeperTestSuite) TestPoolCreationFee() {
 		swaprouterKeeper := suite.App.SwapRouterKeeper
 
 		// set pool creation fee
-		swaprouterKeeper.SetParams(suite.Ctx, swaproutertypes.Params{
+		swaprouterKeeper.SetParams(suite.Ctx, types.Params{
```
Comment (czarcas7ic) [inline x/swaprouter/create_pool_test.go:66]: semi-random aside that doesn't need to be actioned:

I feel like we should have a naming convention for types. IMO I like to call types imports by their name, even the types package of the module I am currently in. The only time I might not do that is if there is only one types imported. What do you think the standard should be?

**Review by czarcas7ic — APPROVED** [review]
Looks good 👍

File: x/swaprouter/create_pool_test.go (hunk 1 above)
Comment (p0mvn) [reply x/swaprouter/create_pool_test.go:66]: My personal convention has been just `types` within a module. E.g., if I'm in `swaprouter`, then swaprotuer's `types` would be `types` but `gamm`'s would be `gammtypes`.

The reason is twofold:
- SDK modules already follow this convention
- Easy to see whether I'm using types of the current module I'm in or an outside module

Happy to discuss separately to come to a consensus

**Comment by p0mvn** [comment]
Merging since this is a test

**Pull request merged**